API_GUIDE.md

```diff
@@ -16,6 +16,8 @@ print(x)
 
 The XLA device is not a physical device but instead stands in for either a Cloud TPU or CPU. The underlying storage for XLA tensors is a contiguous buffer in device memory and the code in the model shouldn't assume any stride.
 
+XLA Tensor doesn't support converting single tensor to half precision using `tensor.half()`, instead environment variable `XLA_USE_BF16` is available to tranforms **all** the PyTorch Float values into BiFloat16 when sending to the TPU device.
```

Comment:
Make this two sentences: "... using `tensor.half()`. Instead, environment variable `XLA_USE_BF16`...".

Reply:
The format is called `bfloat16`, not `BiFloat16`.

Reply:
"... which converts **all** PyTorch float values to bfloat16 when sending them to the TPU device."